<launch>
    <arg name="model" default="$(find mppi_ros)/scripts/mppi_tf/config/models/auv_nn_model.yaml" doc="The internal model used by mppi"/>

    <node name="mppi_learner" pkg="mppi_ros" type="mppi_learner.py" output="screen">
        <rosparam command="load" file="$(arg model)" ns="model"/>
    </node>
</launch>